### DISCUSSION TITLE
2021/07/24/my-fursona/

### DISCUSSION THREAD

**Discussion by giscus** [su226/su226.github.io#8] [Announcements]
# 2021/07/24/my-fursona/

提示本内容已引入版本管理，版本号遵循SemVer（语义化版本）规范，而状态则分为5个：草稿（Draft）、候选（Candidate）、活跃（Active）、弃用（dEprecated）和移除（Removed）。 2021-07-24 原版[A]基础设定 v1.0.0+20210724  主色调：浅蓝 副色调：深灰/白 性格：黏人、非常黏人（doge） 名字：SU226-A01（其实应该算

https://legacy.su226.eu.org/2021/07/24/my-fursona/

**Comment by longzhimenglog** [comment]
从b站来的,贡献第一条评论!(最开始是看格雷科技社区版认识的~)
